Sass/style.css

```diff
@@ -0,0 +1,306 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+@import url('http://fonts.cdnfonts.com/css/lato');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #f9f9f9 !important;
+  position: relative;
+}
+
+body .aboutMLogo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 60px;
+  width: 70%;
+}
+
+body footer img {
+  padding: 20px;
+}
+
+body .logo img {
+  height: 8rem;
+}
+
+body .showUlItems {
+  display: flex !important;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #ec5242;
+  position: fixed;
+  padding-top: 40px;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  margin: 0 !important;
+  overflow-y: hidden;
+  z-index: 100;
+}
+
+body .showUlItems a {
+  color: white !important;
+  font-size: 20px;
+}
+
+body .show {
+  display: flex !important;
+}
+
+body .hide {
+  display: none;
+}
+
+body header {
+  width: 90%;
+  margin: auto;
+}
+
+body header .logo img {
+  height: 4rem;
+}
+
+body header h3 {
+  color: #ec5242;
+  font-size: 24px;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: 300;
+  padding: 25px 0 10px 0;
+}
+
+body header h1 {
+  color: #ec5242;
+  font-weight: 700;
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 3rem;
+  line-height: 1em;
+}
+
+body main {
+  width: 90%;
+  margin: auto;
+}
+
+body main .hero {
+  background-image: url(../img/shape-set-01.png);
+  background-size: 250px;
+  background-position-y: 250px;
+  background-repeat: no-repeat;
+  padding: 2rem 1rem 7rem 1rem;
+}
+
+body main .hero .moment {
+  background-color: #f7f7f7;
+  border: solid 0.25rem white;
+  font-family: 'Lato', sans-serif;
+  margin: 15px 0;
+}
+
+body main .hero .date {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: 900;
+  font-size: 30px;
+}
+
+body main .hero .ejay {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: 800;
+  font-size: 20px;
+}
+
+body main .mainProgram {
+  background-image: url(../img/download.svg);
+  background-color: #272a31;
+  padding: 25px 0;
+}
+
+body main .mainProgram button {
+  width: 92%;
+  margin: auto;
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #ec5242;
+  color: white;
+  padding: 24px 48px;
+  font-weight: 600px;
+  cursor: pointer;
+  align-self: center;
+}
```

Comment:
- [ ] I understand that the previous reviewer asked you to add `font-weight` for this button, but the value you assigned it is an invalid one. Font-weights don't take `px` or any measurement as values. It should be
 `font-weight: 600;`. You can read more on `font-weight` in this [article](https://www.w3schools.com/cssref/pr_font_weight.asp)

Reply:
The right width was assigned